training/training_population.py

```diff
@@ -0,0 +1,65 @@
+from utils import text_extractor
+import validators
+from unittest import TestCase
+
+SIZE_OF_TRAINING_EXAMPLE = 5
+
+
+class WebsiteIterator:
+    def __init__(self, url):
+        self.url = url
+        self.content = self.get_website_text()
+        self.index = 0
+        self.content_length = len(self.content)
+
+    def get_website_text(self):
+        all_text = ''
+        if self.is_website_url(self.url):
+            all_text, title = text_extractor.get_all_text_from_url(self.url)
+        else:
+            all_text = self.url
+        return all_text
+
+    def next(self):
+        lines = self.content.splitlines()
+        copy_from = self.index + SIZE_OF_TRAINING_EXAMPLE
+        if copy_from >= self.content_length:
+            copy_from = self.content_length - self.index
+        if self.index >= self.content_length:
+            raise StopIteration
+        else:
+            result = lines[self.index:copy_from]
+            self.index += SIZE_OF_TRAINING_EXAMPLE
+            return result
+
+    def is_website_url(self, url):
+        if validators.url(url):
+            parsed_url = validators.url(url)
+            if parsed_url.scheme in ['http', 'https'] and parsed_url.hostname:
+                return True
+        return False
+
+
+def get_iterator_for_website(url):
+    return WebsiteIterator(url)
+
+
+class TestTrainingPopulationLog(TestCase):
+
+    def test_next_five_lines(self):
+        text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore\n' \
+               'et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut\n' \
+               'aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in\n' \
+               'voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat\n' \
+               'non proident, sunt in culpa qui officia deserunt mollit anim id est laborum\n' \
+               'Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium,\n'
+
+        iterator = get_iterator_for_website(text)
+        result = iterator.next()
+        assert ['Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore',
+                'et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut',
```

Comment:
@Uriel867 few things: 
1-  you *never* mix test with production code.
as I said before, please follow examples of other existing tests

2- your test has only very basic functionally.
I would expect testing edge cases as well (for example, test next after next, test edge cases like text with only 1 line... be creative and destructive ;)